test/powershell/Modules/Microsoft.PowerShell.Management/Get-PSProvider.Tests.ps1

```diff
@@ -29,6 +29,19 @@ Describe "Get-PSProvider" -Tags "CI" {
         It 'has PathSeparator property' {
             (Get-PSProvider FileSystem).PathSeparator | Should -Be @("\", "/")
         }
+
+        It 'changes on copy do not affect original PathSeparator' {
+            $separator = (Get-PSProvider FileSystem).PathSeparator
```

Comment:
We need tests for all providers. You could use "case" parameter in "It" block.

Reply:
Done. Also fixed the expected values based on the ordering of default path and alternate path separators used from `StringLiterals` class to build the static arrays